app/experimenter/nimbus-ui/src/components/PageHome/Directory.tsx

```diff
@@ -52,7 +61,8 @@ const Directory: React.FunctionComponent = () => {
                 <td>
                   {featureConfig ? featureConfig.slug : "Simple Experiment"}
                 </td>
-                <td>{status}</td>
+                <td>{getStatus(experiment)}</td>
+                <td>{owner.username}</td>
```

Comment:
Not sure if we need an additional check here to make sure old experiments with `null` owners don't throw hard errors here if we try and access a `.username` property on a null object. I don't know if we can do `owner?.username` here, or if we need to wrap it in a `<NotSet/>` component or something.